### PR TITLE
Open Collective

### PR DESCRIPTION
**Pull request by nemchik**
This adds the logo and website links for the top 10 sponsors from Open Collective to the donate page.

Demo: https://lsio.nemplex.com/donate

Edit: There have been changes since the initial PR opened and this is currently a work in progress while an acceptable option is reviewed.

### CODE REVIEW

**Comment by j0nnymoe** [comment]
No issue with this myself. @JAGFin1 to merge if happy.

**Comment by nemchik** [comment]
P.s. the `transferring project balance` transaction here https://opencollective.com/linuxserver/transactions for some reason has lsio (the organization on open collective) considered a sponsor of itself. A core contributor of the collective can contact support to have this adjusted on their back end if you don't want to appear as your own sponsor. The `become a sponsor` image will still show in the sponsor area.

**Comment by aptalca** [comment]
It was the initial balance transfer. One time thing. I don't think it causes an issue

**Comment by nemchik** [comment]
Right, I know what it is. What I'm saying is that if you would like for LSIO (the organization on Open Collective) to not occupy one of the 10 slots for the top financial contributors to your Open Collective you can ask them to adjust this on their end. This really doesn't even become much of a concern until you get 10 sponsors.

**Comment by aptalca** [comment]
Ah, the top donors. Gotcha. I thought it was only the latest.

**Comment by nemchik** [comment]
Also FYI this change only shows the top 10 *sponsors* (open collective tier) by dollar amount. So if someone has been a backer for ages and/or makes custom donations and has contributed let's say $350 in total which would be more than a sponsor that signed up for their first month ($100) the backer will not show up on this list but the new sponsor will.

If that's not the preference and you'd rather have just anybody who has the highest balance contributed I can look into adjusting this.

This relates to my previous comments about lsio being considered its own *sponsor*. It's for some reason flagged as the *sponsor* tier.

**Comment by JoshStark** [comment]
Can this be changed to show either backers and sponsors, or just by highest contributed. Our organisation isn’t really the type to attract a lot of people who are willing to contribute over $100 per month, which is entirely understandable. 

I like the idea of listing our top contributors but limiting it to just sponsors will leave that page rather empty.

**Comment by nemchik** [comment]
I'm pretty sure I can get it to show any donation tier (backer or sponsor). I can make two (or more) sections with one for each tier.

Also I am pretty sure I can just have it show top contributors regardless of tier.

If you're not many (any?) $100/mo sponsors maybe it's a fair idea to lower the amount for a sponsorship?

**Comment by JoshStark** [comment]
I would rather the page not be too loud, so that’s a no on the multiple sections. Having one list with the top ten contributors or ten most recent contributors irrespective of donation amount is fine.

**Comment by nemchik** [comment]
https://github.com/opencollective/opencollective/issues/2593
https://github.com/opencollective/opencollective/issues/2594
https://github.com/opencollective/opencollective/issues/2596
https://github.com/opencollective/opencollective/issues/2597

Not all 100% related, but these are the issues I opened with Open Collective

**Comment by nemchik** [comment]
Updated to use just the donate widget. The issues with this have been mostly resolved (displaying organizations correctly). The `limit` option still does not seem to be working, but the widget still has the best overall presentation at this point.

Ready for review.

**Comment by thelamer** [comment]
@JoshStark this looks good to me

**Comment by JoshStark** [comment]
Sorry this completely skipped my mind. That widget looks much better! Before I approve would it be possible to get a top margin of around 20-30px on the "Financial Contributors" header? It is a little bit too close to the supporter images.

Other than that (admittedly picky thing), it's good from my perspective.

File: donate.html
Comment (JoshStark) [inline donate.html]:
```suggestion
                <div class="col-md-12 text-center mt-5">
```
Adding a bit of margin to the top should help separate this header from the images above.

**Comment by nemchik** [comment]
> Sorry this completely skipped my mind. That widget looks much better! Before I approve would it be possible to get a top margin of around 20-30px on the "Financial Contributors" header? It is a little bit too close to the supporter images.
> 
> Other than that (admittedly picky thing), it's good from my perspective.

Done. Demo updated.

**Review by JoshStark — APPROVED** [review]

**Pull request merged**
